src/Latex/MathLatexSnippet.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mathematicator\Numbers\Latex;
+
+
+use Stringable;
+
+final class MathLatexSnippet implements Stringable
+{
+
+	/** @var string */
+	public $latex;
+
+	/** @var string */
+	private $delimiterLeft;
```

Comment:
property `$delimiterLeft` and `$delimiterRight ` can be null, because it is not defined in the constructor as required.